pkg/util/aad/aad.go

```diff
@@ -74,9 +95,21 @@ func GetToken(ctx context.Context, log *logrus.Entry, clientID, clientSecret, te
 
 		return false, nil
```

Comment:
Hi Spencer, 
No it shouldn't be because we are not posting all the captured errors to customer, only certain errors we are posting to them and for others we send generic error message (Internal Server Error). If we return the error instead of nil, it will directly post the error to customer, which we do not want to do, that is why we are returning nil there instead of err, you can verify the scenario with the existing code in master branch.